package.json

```diff
@@ -1,10 +1,14 @@
 {
   "name": "npm-watcher",
   "version": "1.0.0",
-  "description": "Watches package repo npm feeds (e.g. PyPi and Maven) and pushes incoming changes to a RabbitMQ event queue",
+  "description": "Watches package repo npm feeds and pushes incoming changes to a RabbitMQ event queue",
   "main": "src/server.js",
   "scripts": {
-    "test": "ava"
+    "docker:start": "docker-compose up -d",
+    "docker:stop": "docker-compose down",
+    "start": "node src/server.js",
+    "test": "npm run docker:start && sleep 10 && nyc ava test/unit/*.js && sleep 1 && nyc ava test/integration/*.js && sleep 1 && npm run docker:stop",
```

Comment:
Can you change this to run all tests together? You'll just need to use `nyc ava test` instead and remove the `sleep 1`.

Also, add extra commands for just running unit tests and integration tests separately. 
```sh
# run unit tests only
npm run test:unit

# run integration tests only
npm run test:integration
```